src-tauri/src/lib.rs

```diff
@@ -239,12 +239,26 @@ pub async fn install_northstar(
         ..Default::default()
     });
 
-    do_install(
+    match do_install(
         nmod.versions.get(&nmod.latest).unwrap(),
         std::path::Path::new(game_path),
     )
     .await
-    .unwrap();
+    {
+        Ok(_) => (),
+        Err(err) => {
+            if game_path
+                .to_lowercase()
+                .contains(&r#"C:\Program Files\EA Games\Titanfall2"#.to_lowercase())
```

Comment:
Can the EA App games path be different from this one?

Reply:
Why not checking for `C:\Program Files\` only then?

Reply:
Fair enough, changed in 8992810e0fcce01ba9b7b0c480db8d80009e1f24